WultraMobileTokenSDK/Operations/Model/UserOperation/Screens/WMTPostApprovaScreenReview.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2023 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+public class WMTPostApprovalScreenReview: WMTPostApprovalScreen {
```

Comment:
According to backend implementation, the `REVIEW` type can have payload with operation attributes - see https://github.com/wultra/enrollment-server-wultra/blob/57fbdade37c3f15e25a5e18b78acd4228f9f0d3f/docs/Operation-Extensions.md?plain=1#L83